docs/user_guide/optimization.md

```diff
@@ -83,7 +83,9 @@ For most models, the Triton feature that provides the largest
 performance improvement is [dynamic
 batching](model_configuration.md#dynamic-batcher). If your model does not
 support batching then you can skip ahead to [Model
-Instances](#model-instances).
+Instances](#model-instances). 
+[This example](https://github.com/triton-inference-server/tutorials/tree/main/Conceptual_Guide/Part_2-improving_resource_utilization#dynamic-batching--concurrent-model-execution)
```

Comment:
Put this before the model instance sentence. i.e.

---

For most models, the Triton feature that provides the largest performance improvement is [dynamic batching](model_configuration.md#dynamic-batcher). [This example](https://github.com/triton-inference-server/tutorials/tree/main/Conceptual_Guide/Part_2-improving_resource_utilization#dynamic-batching--concurrent-model-execution) goes into depth on the conceptual details.

If your model does not support batching then you can skip ahead to [Model Instances](#model-instances).